xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml

```diff
@@ -644,7 +644,7 @@ Get the actual notifications.
       <name>LESS</name>
     </property>
     <property>
-      <parse>1</parse>
+      <parse>0</parse>
```

Comment:
The CSS above still contains Velocity code, how is it okay to disable parsing here?